.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: haskell
+
+ghc:
+  - 7.8
+  - 7.6
+  - 7.4
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install haskell-platform g++ cmake pkg-config libzip-dev libopenal-dev libsndfile1-dev
+
+install:
+  - cd src
+  - ./linuxCompile.sh --extra-include-dir=/usr/include/AL --constraint=bindings-libzip==0.10.*
+  - cabal install -j --only-dependencies --constraint=bindings-libzip==0.10.* --extra-include-dir=/usr/include/AL && cabal configure --enable-tests && cabal build && cabal test
```

Comment:
`linuxCompile` already calls `cabal` so this builds `nikki` twice. Maybe we can just call `cabal test` after `linuxCompile`. (We have to pass `--enable-tests` to it then, I guess.)